aii-opennebula/src/main/perl/opennebula.pod

```diff
@@ -116,6 +118,40 @@ Following package dependencies should be installed to run the AII:
 
 =back
 
+Set OpenNebula endpoints RPC connector /etc/aii/opennebula.conf
+
+It must include at least one RPC endpoint and password.
+
+By default ONE AII uses oneadmin user.
+
+It is also possible to set a different endpoint for each VM domain, as example:
+
+=over 
```

Comment:
don't use a pod list here, just a raw text field. indent eacch line with 4 spaces and this should work